omnigan/transforms.py

```diff
@@ -81,6 +83,85 @@ def __call__(self, data):
         return {task: torch.flip(tensor, [3]) for task, tensor in data.items()}
 
 
+class RandomRotations:
+    def __init__(self, p=0.5, angle=5):
```

Comment:
This needs way more comments and docstrings explaining your logic for  the PR to be mergeable.

Especially (but not only): `seg_dict`, `mapping(...)`, difference between `s` and other tasks, why go to tensor then to pil then to numpy then back to tensor and what not